Packs/FeedTAXII/Integrations/FeedTAXII/FeedTAXII_test.py

```diff
@@ -98,6 +98,16 @@ def test_fetch_indicators(self, mocker):
                 assert res == expected
 
 
+def test_poll_collection():
+    import requests_mock
+    from FeedTAXII import TAXIIClient
+    client = TAXIIClient(collection='a collection')
+    with requests_mock.Mocker() as m:
+        m.post('google.com')
+        res = client._poll_collection('http://google.com', '1', '2')
+    assert res
+
+
```

Comment:
It looks like you are almost not testing the code you added.
Please go over it and make sure to add relevant tests.